### PR TITLE
[Fix] csv api endpoint path

### PR DESCRIPTION
**Pull request by dishantsethi**
The CSV row is redirecting to ```radiusbatch/csv/<uuid>.csv``` which is an incorrect URL.
Instead, it should be ```/api/v1/radiusbatch/csv/<uuid>.csv```

### CODE REVIEW

**Comment by dishantsethi** [comment]
Changed Download batch CSV API endpoint from ```radiusbatch/csv/<uuid>/filename.csv``` to ```radius/organization/<slug:slug>/batch/<uuid:pk>/csv/filename.csv``` maintain consistency.

@nemesisdesign @atb00ker  Review Required!

I'll be adding this is swagger API docs post review.

File: docs/source/user/api.rst [hunk 1]
```diff
@@ -694,6 +694,25 @@ Param              Description
...
+
+Responds only to **GET**.
+
+Parameters:
+
+===============    ===============================
+Param              Description
+===============    ===============================
+slug               string
+id               string
```
Comment (atb00ker) [inline docs/source/user/api.rst:712]: Please check spacing. (tab or spaces) here.

**Review by atb00ker — COMMENTED** [review]
Minor problem, see below.
The rest seems good! 😄

**Comment by dishantsethi** [comment]
@atb00ker Fixed!

**Comment by Aryamanz29** [comment]
> @atb00ker Fixed!

Please run following commands on terminal :

QA commands 👇
```bash
openwisp-qa-format
./run-qa-checks
```
Test commands :point_down: 
```bash
coverage run --source=openwisp_radius runtests.py
SAMPLE_APP=1 ./runtests.py --parallel --keepdb
```
Also follow `openwisp commit message guidelines` : http://openwisp.io/docs/developer/contributing.html#commit-message-style-guidelines

File: openwisp_radius/settings.py [hunk 2]
```diff
@@ -131,7 +131,7 @@ def get_settings_value(option, default):
     )
 
 # Path of urls that need to be refered in migrations files.
-CSV_URL_PATH = 'api/v1/radiusbatch/csv/'
+CSV_URL_PATH = 'api/v1/radius/organisation/'
```
Comment (codesankalp) [inline openwisp_radius/settings.py:134]: For consistency:
```suggestion
CSV_URL_PATH = 'api/v1/radius/organization/'
```

**Review by codesankalp — CHANGES_REQUESTED** [review]

**Comment by dishantsethi** [comment]
Hi @atb00ker @codesankalp 
Made the necessary changes and ran tests locally. Let me know if anything is left.

**Review by atb00ker — COMMENTED** [review]
Hey commit guide is still wrong, 
Was `openwisp-qa-checks` passing on your local? (That might be some setup issue!)

To fix it, do:
```
$ git commit --amend
#
[change] Update download batch CSV URL and add in API docs #373

Closes #373
```

**Comment by dishantsethi** [comment]
Updated the commit message. @atb00ker

**Review by atb00ker — DISMISSED** [review]

**Review by nemesisdesign — APPROVED** [review]
Thanks :+1:

**Pull request merged**